blog/2024-03-20-typescript-client-beta/index.mdx

```diff
@@ -328,146 +331,71 @@ We won't get into the details of gRPC here, but if you are interested, you can r
 
 ## Object typing in focus
 
-[What example can we give?]
+TypeScript is a fully-featured strongly-typed language that comes with a plethora of features for the manipulation of user-defined types. This has allowed us to design and create an API where your specific types can merge with our general Weaviate types to enhance user experience and developer efficieny. Even when writing some of the tests, the IDE auto-completion and static type checking provided by the API have been immensely helpful!
 
 > **Highlights: Strong typing in queries, retrieval augmented generation, data operations, ...**
 
-We've mentioned object typing a few times already, but let's take a closer look at a few more concrete examples. Here you can see the new syntax for collection definition, queries and retrieval augmented generation, as well as types for each of thees tasks.
-
-To get started you can import the set of submodules like shown below, and use them as needed.
+We've mentioned object typing a few times already, but let's take a closer look at a few more concrete examples. Here you can see the new syntax for collection definition, queries and retrieval augmented generation, as well as types for each of these tasks.
 
+All the types of the API are exported from the root file. So, when it comes to using any generic, you can access them all like so:
 ```typescript
-import weaviate.classes as wvc  # <-- import the custom classes
-```
+import weaviate, { Collection } from 'weaviate-client';
 
-[whats the equivalent? Default exports?]
+const collection: Collection = client.collections.get("MyCollection");
+```
 
-### Collection definition
```

Comment:
you completely took out the collections definition section, don't think we should have it?

Reply:
Ah, my mistake! I meant to refactor and add it back but didn't before opening the PR. Will add now 😁

Reply:
ok, cool!